sql/telemetry/fxa_content_auth_events_v1/view.sql

```diff
@@ -6,6 +6,7 @@ WITH
   SELECT
     jsonPayload.logger,
     jsonPayload.fields.event_type,
+    jsonPayload.fields.app_version,
```

Comment:
We include app_version in the existing fxa_users_daily, etc. queries. Seems we might as well keep that dimension, so need to include in the views here.